tests/modules/js-and-css/optimize-embeds/optimize-oembeds-test.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Tests for embed optimizer.
+ *
+ * @package performance-lab
+ * @group embed-optimizer
+ */
+
+class Embed_Helper_Tests extends WP_UnitTestCase {
+
+	/**
+	 * Test that the oEmbed HTML is filtered.
+	 *
+	 * @dataProvider test_oembed_optimize_html_data
+	 */
+	public function test_perflab_optimize_oembed_html( $html, $expected ) {
+		$this->assertEquals( $expected, perflab_optimize_oembed_html( $html ) );
```

Comment:
Instead of having the `perflab` prefix, I think this should have a plugin-specific one, right?

Reply:
right

Reply:
fixed